src/containers/SearchPage/components/results/SearchConcept/ConceptForm.tsx

```diff
@@ -163,29 +178,15 @@ const ConceptForm = ({ initialValues, status, language, onSubmit, allSubjects, c
         </Button>
         <MultiButton
           disabled={!hasChanges || Object.keys(errors).length > 0}
-          className="form-button"
-          onClick={(value: string) => {
-            const getStatus = (v: string, s: string) => {
-              if (v === 'saveAndPublish') {
-                return 'PUBLISHED';
-              } else if (s === 'PUBLISHED') {
-                return 'QUALITY_ASSURED';
-              } else {
-                return undefined;
-              }
-            };
-            const newStatus = getStatus(value, status);
-            onSubmit({ ...values, newStatus });
-          }}
-          mainButton={{ value: 'save' }}
+          onClick={value => handleClick(value)}
+          mainButton={{ value: 'save', label: t(`form.save`) }}
           secondaryButtons={[
             {
               label: t('form.saveAndPublish'),
               value: 'saveAndPublish',
             },
-          ]}>
-          <span>{t(`form.save`)}</span>
-        </MultiButton>
+          ]}
```

Comment:
Var det meningen å fjerne `span` her?

Reply:
Ja, det er en conditional check inn i `<MultiButton/>` komponenten som sjekker om children er passert og hvis så, bruker den de, hvis ikke bruker den label verdien. Så jeg bare endte opp med å bruke label sin verdi, no need to sende hele span verdien her når man får samme resultatet med å sette `label` til samme verdi.